CMakeLists.txt

```diff
@@ -172,37 +172,10 @@ endif(USE_STATIC_LIBS)
   # Cmake. This is also fairly untested, let's hope it does not explode.
```

Comment:
Is this remark still applicable? It seems to work now, no?

Reply:
True, that can be removed.